docs/00-manuscripts/ieee/reviews/reviewer4.md

```diff
@@ -78,9 +80,11 @@ We added this information to the section describing the in-situ data.
 Thanks for pointing this out.
 It is true that the use of a buffer (or in fact the existence of a geometric offset) provides discussion potential in this work.
 There were long discussions among the authors if and how to tackle this problem.
-Not doing anything would have pushed the risk of including pixels without any tree information at all (e.g. bare ground).
-Including such false positives would have an substantial influence for the model due to the apparent difference of bare ground pixels compared to vegetation pixels.
-Therefore we concluded to use a buffer and by this to average the values of the neighboring pixels of each tree observation.
+<!--
+Not doing anything would have increased the risk of including pixels without any tree information at all (e.g. bare ground).
+Including such false positives would have a substantial influence for the model due to the strong difference of bare ground pixels compared to vegetation pixels.
+-->
```

Comment:
Too specific? I thought that arguing with the "bare ground fear" could be convincable.